lib/knapsack_pro/adapters/base_adapter.rb

```diff
@@ -4,6 +4,14 @@ class BaseAdapter
       # Just example, please overwrite constant in subclass
       TEST_DIR_PATTERN = 'test/**{,/*/**}/*_test.rb'
 
+      def self.slow_test_file?(path)
+        @slow_test_file_paths ||= KnapsackPro::TestFileFinder.call(
```

Comment:
TODO: Read list of slow test files fetched from API based on existing file on the disk created by lib/knapsack_pro/test_case_detectors/rspec_test_example_detector.rb